src/Google/Ads/GoogleAds/Lib/V1/LogMessageFormatter.php

```diff
@@ -136,7 +156,9 @@ public function formatDetail(
             ) {
                 self::$unusedGoogleAdsFailure = new GoogleAdsFailure();
             }
-            $logMessageTokens[] = "Response: {$response->serializeToJsonString()}";
+            $logMessageTokens[] = "Response: " . (is_null($response) ? "None" :
```

Comment:
I agree it is better to make it consistent with V3 implementation. No issue at all but please note that this should only be useful in the context of ServerStream (never in V1).

Reply:
At first, I wanted to update only V3, but I guess you will ask me to update v2 too, so I ended up updating all versions. :)